(maint) add LICENSE file

melissa opened the pull request.

melissa committed:
(maint) add LICENSE file

pmcmaw commented:
Change LGTM 👍 

The following fixes have went into the pdk-templates which will resolve your travis issue: 
https://github.com/puppetlabs/pdk-templates/pull/171/
https://github.com/puppetlabs/pdk-templates/pull/173

These have not been released yet as part in built template fo the pdk. It is possible to convert your modules to take changes from the live repo rather than having to wait for a pdk release. If this is something yous would be interested in, to avoid blockers waiting on template changes feel free to ping in the modules room and someone will be more than happy to assist. We use a tool called pdksync to roll out bulk updates to all of our supported modules and this may be helpful for your teams supported modules. :-)

melissa commented:
I updated the pdk templates for all core modules yesterday afternoon, and most have been merged. I just need to rebase these license PRs :) This one is now updated with the latest updates in master, and should pass!
